### ISSUE TITLE
There was a problem with the results of the FRN test

### ISSUE THREAD

**Issue by wen0320** [Crystalsound/FRN#5]
First of all，I would like to express my deepest gratitude to the author for open-sourcing such an excellent model.
I tested the performance of the FRN packet loss compensation using a test machine from the 2022 PLC Challenge (using a pre-trained model published by the authors on hugging face), and the final result was PLCMOS: 3.21. It's not what I expected. I think PLCMOS should be at least 3.7.So I am skeptical about this result。

**Comment by vietanh125** [comment]
Hi, thank you for your interest in our work. There's one small note to reproduce the results on the 2022 PLC Challenge test set. As mentioned in section 3.1, we only used the loss traces from the dataset and apply them on the test audio again. This is because we observed that the lossy segments in the test audios haven't been zeroed out completely, and our model are sensitive to these artifacts. Since you are applying the model directly on the test audios, a performance degradation is expected. Hope it helps.
